Misc/NEWS.d/next/Core and Builtins/2019-07-08-17-11-20.bpo-37187.-KLvmv.rst

```diff
@@ -0,0 +1 @@
+Print bit width in :c:func:`PyCField_get_size`
```

Comment:
I believe `c:func` should only be used for the documented functions of the C API.

Reply:
en, looks somebody use this tag too. I am not sure about it.
https://github.com/python/cpython/blob/master/Misc/NEWS.d/next/Core%20and%20Builtins/2019-06-17-03-53-16.bpo-37316.LytDX_.rst

Reply:
Zackery is correct, the tag is only for documented functions (the other example - `PySys_Audit` - is documented).
```suggestion
Return bit width in ``PyCField_get_size``
```

Reply:
Thanks for review, Steve. Done ;)